14-Jenkins/base.yaml

```diff
@@ -14,13 +14,18 @@ Parameters:
   EC2InstanceAmi:
     Type: AWS::EC2::Image::Id
     Description: EC2 Instance AMI
-    Default: ami-0bce08e823ed38bdd
+    Default: ami-014534a11156c1a0d
```

Comment:
Is this a public image? If so, maybe we could update to a SSM parameter. 

Similar to https://github.com/stelligent/stelligent-u/pull/23/files

Reply:
It is public. I can do that.